runIntegration.sh

```diff
@@ -177,7 +177,7 @@ cd $CURWD
 # Step 3) Get the base containers running
 echo "Starting containers..."
 ${docker_compose_exe} pull \
-    && ${docker_compose_exe} build --parallel \
```

Comment:
why did you drop parallel?

Reply:
Older versions of docker-compose don't support it.  The build time is pretty fast even
without it.